tests/drivers/build_all/gpio/testcase.yaml

```diff
@@ -6,3 +6,9 @@ tests:
     min_ram: 32
     platform_exclude: serpente
     depends_on: gpio spi i2c
+
+  drivers.gpio.build.altera_pio:
+    min_ram: 32
+    platform_allow: niosv_m
+    depends_on: gpio spi i2c
```

Comment:
Is this depends line right? Also, should this be build-only?

Reply:
We are not sure, we copied it from the other test.